src/pls/table.py

```diff
@@ -6,32 +6,53 @@
 from pls.args import args
 from pls.enums.icon_type import IconType
 from pls.models.node import Node
-
+from sys import platform
 
 console = Console()
-
-column_spec = {
-    "": {"name": ""},  # dummy column to act like spacer
-    "type": {
-        # 'type' is a pseudo-column linked to 'perms', so it has no name.
-        "name": ""
-    },
-    "perms": {"name": "Permissions"},
-    "user": {"name": "User"},
-    "group": {"name": "Group"},
-    "size": {"name": "Size", "attrs": {"justify": "right"}},
-    "icon": {
-        # 'icon' is a pseudo-column linked to 'name', so it has no name.
-        "name": "",
-        "attrs": {"width": 2},
-    },
-    "name": {
-        # The names have a leading space when the leading dots are aligned.
-        "name": "Name"
-        if args.no_align
-        else " Name"
-    },
-}
+if platform != "win32":
```

Comment:
There is no need to conditionally define the `column_spec` variable. Just removing the user and group column from the table should sufice.